sugarscape.py

```diff
@@ -960,9 +963,22 @@ def verifyConfiguration(configuration):
         configuration["startingAgents"] = totalCells
 
     # Ensure infinitely-lived agents are properly initialized
-    if configuration["agentMaxAge"][0] < 0 or configuration["agentMaxAge"][1] < 0:
-        configuration["agentMaxAge"][0] = -1
-        configuration["agentMaxAge"][1] = -1
+    if configuration["agentMaxAge"][0] < 0:
+        configuration["agentMaxAge"] = [-1, -1]
+
+    if configuration["agentSelfishnessFactor"][0] < 0 and configuration["agentSelfishnessFactor"][1] >= 0:
+        if "all" in configuration["debugMode"] or "agent" in configuration["debugMode"]:
+            print(f"Cannot have agent selfishness factor range of {configuration['agentSelfishnessFactor']}-{configuration['agentSelfishnessFactor'][1]}. Disabling 'agentSelfishnessFactor'.")
```

Comment:
This is a minor style nitpick, but we generally use double quotes for accessing keys.